Workshop

przmv opened the pull request:
This PR adds the actual workshop materials

TODO:

- [ ] Use `main` branch in **Open in Cloud Shell** button in `README.md`

Commit:
Initial commit

Commit:
Update README.md

Commit:
Fix newlines at EOF

SamDesmondKing reviewed (commented):
Looks awesome Petr. One small suggestion, maybe you could add the command to add the machine stats script location to the PATH? 

![image](https://user-images.githubusercontent.com/43866616/129047110-1e28eada-4ad0-4601-8739-06b472af927e.png)

jmariomejiap reviewed (approved):
It looks good to me @przmv. It is nice to get this workshop content ready for future uses 👍

przmv commented:
On Wed, 2021-08-11 at 07:25 -0700, Sam King wrote:
> @SamDesmondKing commented on this pull request.
> Looks awesome Petr. One small suggestion, maybe you could add the
> command to add the machine stats script location to the PATH?

Thanks, Sam!

Actually, it should be added. I'll check.

Commit:
Remove package manager cache

Commit:
No need for additional checks

No need for additional checks, `bashrc.google` is sourced only once

Commit:
Remove `apt-get update`

Commit:
Open only shell

Commit:
Install ansible from pip

See https://github.com/tidalmigrations/machine_stats/issues/71

Commit:
remove walkthrough-enable-apis

walkthrough-enable-apis only works in Cloud Console, not Cloud Shell

Commit:
No need to explicitly open shell

Commit:
Adding deployment manager configuration

Adding deployment manager configuration to create a pool of VMs

Commit:
Use deployment manager

Use Google Cloud Deployment Manager instead of interactive VMs creation,
because it doesn't work in Cloud Shell

Commit:
Use script instead of long command

Commit:
Open Cloud Shell

Open Cloud Shell with the current project selected

Commit:
Set the current project within the Cloud Shell

Commit:
Enable APIs

Commit:
Rename to machine-stats.cfg

Commit:
Remove footnote about cron and systemd

Commit:
Run repeated syncs for five times

Run repeated syncs for five times instead of endleess loop

Commit:
Remove redundant links

Commit:
Charges and clean up notes

jmariomejiap commented on machine-stats.md:
````diff
@@ -0,0 +1,227 @@
+in this tutorial, follow this steps.
+
+```bash
+gcloud deployment-manager deployments delete instance-pool
````
Nice, 👍

jmariomejiap commented on machine-stats.md:
```diff
@@ -0,0 +1,227 @@
+Alternatively, you can delete your Cloud project to stop billing for all the
+resources used within that project.
+
+**Caution**: Deleting a project has the following effects:
```
good idea to add this extra section 👍

jmariomejiap reviewed (approved):
now, It's even better. It looks ready to be released @przmv 🚀

surmistry commented:
Just ran through it again 👍🏽 
Great job making a good flow into a seamless, clean experience

Commit:
Use default git branch
